src/plugins/numberline/numberline-tile.tsx

```diff
@@ -1,14 +1,92 @@
 import { observer } from "mobx-react";
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
 import { ITileProps } from "../../components/tiles/tile-component";
+import { scaleLinear, select, selectAll, pointer, axisBottom } from "d3";
+import { tickWidthDefault, tickWidthZero, tickHeightDefault, tickHeightZero,
+  tickStyleDefault, tickStyleZero, kContainerWidth, kAxisWidth, numberlineDomainMax, numberlineDomainMin } from "./numberline-tile-constants";
 
 import "./numberline-tile.scss";
 
 export const NumberlineToolComponent: React.FC<ITileProps> = observer((props) => {
+  //---------------- Create unique className for tile ------
+  const tileId = props.model.id;
+  const axisClass = "axis-" + tileId;
```

Comment:
One thing I've realized since we discussed this id is that it's possible for the tile to exist in both a read only document on the left and a read/write document on the right. It's worth checking to make sure this doesn't cause a problem in that case. If it does, it might be possible to fix it by incorporating `readOnly` into `axisClass`.